code-challenges.test.js

```diff
@@ -34,26 +49,122 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
-
-// b) Create the function that makes the test pass.
+// psuedo:
+// Create a the funciton divByThree that we established in the test.
+// Give the funciton the parameter object for clarity.
+// Use the arrow funciton syntex.
+// Create a condtional using an if statement to test if the obeject is divisible by three, which will be set to strictly equal to 0.
+// Create an else statement as a catch all for all other numbers that do not meet the criteria above.
+// Set both the if and else statements to return a string interpolated message ' object.number is divisible by three, or object.numer is not divisible by three.
+// DO NOT include a console.log because it is being tested!!!
+
+const divBYThree = (object) => {
+    if(object.number % 3 === 0){
```

Comment:
This is a great approach! When writing your psuedo be I encourage you to be  as detailed with you plan as possible. If you plan on using the modulo, put it in your plan and explain why.